content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+- :guilabel:`Visibility`: use the drop-down menu to add any visibility conditions for this field, if
+  desired.
+
+.. image:: opportunities_form/form-submitted-successfully.png
+   :align: center
+   :alt: The screen displayed after a customer successfully submits a contact form.
+
+.. important::
+   If *leads* are activated in CRM settings, selecting :guilabel:`Create an Opportunity` generates a
+   lead instead. To activate leads, navigate to :menuselection:`CRM app --> Configuration -->
+   Settings` and tick the :guilabel:`Leads` checkbox, then click :guilabel:`Save`.
+
+Customize contact form fields
+-----------------------------
+
+In addition to the settings for the form, the settings for each field can be customized as well.
+With the web editor menu still open, click into a field to open the field configuration settings.
+The following options are available to customize a field:
+
+- :guilabel:`Type`: choose a custom field option or an existing field type.
+- :guilabel:`Input Type`: determine the type of information customers should input. Available
+  options are :guilabel:`Text`, :guilabel:`Email`, :guilabel:`Telephone`, or :guilabel:`Url`. The
+  selection made in this field limits the format that customers can use when entering information.
+- :guilabel:`Label`: enter the name for the field.
+- :guilabel:`Position`: choose the way the label is aligned with the rest of the form. The label can
+  be hidden, above the field, to the left of the field, or right adjusted and closer to the field.
+- :guilabel:`Description`: slide the toggle to add a description for the field, which can provide
+  additional instructions to customers. Click under the field on the form to add the description.
+- :guilabel:`Placeholder`: enter an example to guide users how to input information where
+  formatting is important, such as a phone number or email address.
+- :guilabel:`Default Value`: enter a value to include in the form by default if the customer does
+  not provide information in the field. *It is not recommended to include a default value for
+  required fields*.
+- :guilabel:`Required`: slide the toggle to mark this field as required if it **must** be filled in
+  for every submission.
+- :guilabel:`Visibility`: select when this field should be visible.
+- :guilabel:`Animation`: select if this field should have any animation.
+
+.. image:: opportunities_form/field-customization.png
+   :align: center
+   :alt: The field configuration settings on an Odoo website.
+
+View opportunities
+==================
+
+After a customer submits a contact form and an opportunity is created, it is assigned based on the
+:ref:`form settings <crm/customize-contact-form>`. To view opportunities, navigate to
+:menuselection:`'CRM app' --> Sales --> My Pipeline`.
+
+.. note::
+   If leads are activated on the database, contact form submissions are generated as leads, not
```

Comment:
is the default Opportunities, and they must activat leads? Just curious, ecause that coudl be adde-d I honestly am not sure!

Reply:
Opportunities are the default, leads need to be activated. I will link to the admonition block where this is called out